### PR TITLE
Remove unused UCAS code

### PR DESCRIPTION
**Pull request by davidgisbey**
## Context

I missed a few things while i was pulling out the code. Apologies!

## Changes proposed in this pull request

- Remove unused UCAS content from the codebase

## Guidance to review

Have i missed anything key? There's some content in the support UI about whether a course was open on apply etc., but i thought it might need to stay in as it would still apply to courses open last cycle?

## Things to check

- [x] This code does not rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

File: app/services/feature_flag.rb [hunk 1]
```diff
@@ -13,7 +13,6 @@ def feature
   end
 
   PERMANENT_SETTINGS = [
-    [:banner_for_ucas_downtime, 'Displays a banner to notify users that UCAS is having problems', 'Apply team'],
```
Comment (duncanjbrown) [inline app/services/feature_flag.rb:16]: !

**Review by duncanjbrown — APPROVED** [review]
:+1: LGTM

File: app/services/feature_flag.rb (hunk 1 above)
Comment (davidgisbey) [reply app/services/feature_flag.rb:16]: ?

Comment (davidgisbey) [reply app/services/feature_flag.rb:16]: never again haha

**Pull request merged**